packages/shared/src/errors/Error.ts

```diff
@@ -178,6 +182,7 @@ export const MagicLinkErrorCode = {
   Expired: 'expired',
   Failed: 'failed',
 };
+// TODO(@dimkl): Add deprecation warning for MagicLinkErrorCode
```

Comment:
@dimkl  will this be part of a followup PR ?

Reply:
i will update the current PR. I am trying to find if we could avoid adding a deprecation warning for this (eg it may not be exported or there is no real usage outside of our own SDKs)

Reply:
@panteliselef i've added a deprecation warning for `MagicLinkErrorCode` since it may be used by our customers